lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -17,7 +17,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user why and how they should secure cross-origin links. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Add `rel="noopener"` or `rel="noreferrer"` to any external links to improve ' +
     'performance and prevent security vulnerabilities. ' +
-    '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/noopener).',
+    '[Learn more](https://web.dev/external-anchors-use-rel-noopener).',
```

Comment:
FWIW this doc just came up in #9531 about possibly needing some tweaks

Reply:
I've added this comment to [web.dev issue 1342](https://github.com/GoogleChrome/web.dev/issues/1342) for tracking.